packages/grid/data-grid/src/tests/layout.DataGrid.test.tsx

```diff
@@ -373,6 +390,17 @@ describe('<DataGrid /> - Layout & Warnings', () => {
         ).to.equal(200 - 2);
       });
     });
+
+    it('Grid wrapper should have height different from 0 when autoHeight', () => {
```

Comment:
Maybe we can improve the description now. It's more than different from zero. 
```suggestion
    it('should have the correct intrinsic height when autoHeight={true}', () => {
```